File: nodejs/response_string.js
const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
});

readline.question("Enter your preferred technology at Elktech (Python, NodeJS, Go, Rust): ", tech => {
    console.log("Great choice! " + tech + " is widely used in Elktech projects ");
    readline.close();
});

File: rust/response_string.rs


use std::io;

fn main() {
    let mut tech = String::new();
    println!("Enter your preferred technology at Elktech (Python, NodeJS, Go, Rust): ");
    io::stdin().read_line(&mut tech).expect("Failed to read input");
    let tech = tech.trim();

    println!("Great choice! {} is widely used in Elktech projects .", tech);
}
